### PR TITLE
GridItem Drag Handle

### PR DESCRIPTION
**Pull request by ThePlastic**

### CODE REVIEW

**Comment by ThePlastic** [comment]
related to this:
https://github.com/jbaysolutions/vue-grid-layout/issues/16

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "author": "Gustavo Santos (JBay Solutions) <gustavo.santos@jbaysolutions.com> (http://www.jbaysolutions.com)",
   "dependencies": {
     "element-resize-detector": "^1.1.10",
-    "interact.js": "^1.2.8"
+    "interact.js": "git://github.com/taye/interact.js.git#master"
```
Comment (gmsa) [inline package.json]: Are you using an unreleased interact.js version? That doesn't seem like a good idea...

Comment (ThePlastic) [inline package.json]: You are right, but they are a bit slow on developing, that feature was already ready months ago and I waited for them to publish on npm as I needed it for my project. Unfortunately, it has not been published yet (they have released version 1.3.0alpha which is in the unstable branch and is not available through package.json). So I decided to implement it through the master branch. Feel free to not commit my changes, as soon as the new version of the interactjs library will be published you just have to edit your package.json

Comment (gmsa) [inline package.json]: Ok, if they are that slow I will review this PR and merge it when I have some free time.

Comment (ThePlastic) [inline package.json]: Or if you like you can put 
 `"interactjs": "^1.3.0-alpha.1"`
as a dependency but you have to require it from GridItem.vue like this:
   `var interact = require("interactjs/dist/interact.js");`
and it works like a charm :)
let me know if you need anorther PR

**Comment by timstermatic** [comment]
How do you specify a drag handle, say I only want to be able to drag a GridItem by a specific area?

**Comment by gmsa** [comment]
Check out this example: https://jbaysolutions.github.io/vue-grid-layout/examples/04-allow-ignore.html
You can view the source and css to check how it's done.

**Comment by timstermatic** [comment]
@gmsa that's perfect. Thank you!

**Pull request merged**